.circleci/config.yml

```diff
@@ -73,7 +73,7 @@ jobs:
             - checkout
             - run: sudo pip install .[sklearn,torch,testing]
             - run: sudo pip install -r examples/requirements.txt
-            - run: python -m pytest -n 8 --dist=loadfile -s ./examples/ | tee output.txt
+            - run: python -m pytest -n 8 --dist=loadfile -s ./examples/ --ignore=examples/test_xla_examples.py | tee output.txt
```

Comment:
you could also make a `require_xla` decorator.

Reply:
Very good idea!